scripts/check-semconv-version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+declare -A semconv_map
+
+while IFS=: read -r file_name package_string; do
+    version_number=$(echo "$package_string" | grep -o -E "v1\.[0-9]+\.[0-9]+")
```

Comment:
```suggestion
    version_number=$(echo "$package_string" | grep -o -E "v[0-9]\.[0-9]+\.[0-9]+")
```

Reply:
also, this regex is used more then once, let's move it to a variable

Reply:
also, why do you need this grep here in the first place, isn't it guaranteed to match because of L8?